src/validate.js

```
@@ -28,29 +34,499 @@ for (const [key, value] of Object.entries(schemas)) {
+      transformedObject.loadVariables = object.path;
+    } else if (currentSchema === "typeKeys_v2") {
+      transformedObject.type = {
+        keys: object.keys,
+        inputDelay: object.delay,
+      };
+    } else if (currentSchema === "saveScreenshot_v2") {
+      transformedObject.screenshot = {
+        path: object.path,
+        directory: object.directory,
+        maxVariation: object.maxVariation / 100,
+        overwrite: (object.overwrite = "byVariation"
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)